app/helpers/word_helper.rb

```diff
@@ -0,0 +1,24 @@
+module WordHelper
```

Comment:
These are view helpers that just help with turning numbers (from enums) into words